.pre-commit-config.yaml

```diff
@@ -1,21 +1,24 @@
 repos:
+
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.1.0
     hooks:
       - id: check-added-large-files
       - id: check-json
       - id: check-yaml
-      - id: detect-aws-credentials
+#      - id: detect-aws-credentials
       - id: detect-private-key
       - id: end-of-file-fixer
       - id: trailing-whitespace
-      - id: no-commit-to-branch
-        args: [ --branch, main]
+#      - id: no-commit-to-branch
+#        args: [ --branch, main, --branch, master, --pattern, release/.* ]
+
```

Comment:
Just curious: any particular reason why you took those out?

Reply:
Ah DevOops! Fixing

Reply:
And the aws creds check?

Reply:
Yep; that `id: detect-aws-credentials` hook won't work for SSO-ed CDK. Reproducible: try `rm ~/.aws/credentials` and `make check`. Hence, Yelp `detect-secrets` which cover that plus more heuristic scan vectors.

Reply:
i.e. `cd orcabus && detect-secrets scan --list-all-plugins`

Reply:
Actually, my memory comes back now; why I commented `no-commit-to-branch` hook initially. For this, to work, we either 

A. Better should have `dev` branch as default upstream. 
B. Otherwise that hook scanning hit to the local `main` & block (which is not intended)

Perhaps, we better just wish protect direct push to remote i.e. better be, if we rather just want to stick with a `single` branch strategy.

```
      - id: no-commit-to-branch
        args: [ --branch, origin/main, --branch, origin/master, --pattern, origin/release/.* ]
```